spec/features/organizations/user_can_create_organization_spec.rb

```diff
@@ -6,21 +6,82 @@
   So that we can benefit from helpful volunteers
 } do
 
-  # [ ] Organizer can add their organization’s profile page using a form
-  # [ ] Organizer is then the owner of the organization
-  # [ ] Organizations must have a name, address, email,
+  # [X] Organizer can add their organization’s profile page using a form
+  # [X] Organizer is then the owner of the organization
+  # [X] Organizations must have a name, address, email,
   #     delivery instructions, and description.
   #     URL, phone number, hours, description,
   #     and profile/organization photo are optional.
-
+  # [X] If an organization is not created, the organizer is not added.
   scenario 'logged in user can navigate to a new organization form' do
     user = FactoryGirl.create(:user)
 
     sign_in_as(user)
 
     visit '/'
-    click_link("I am... an organizer, and I want to add my organization.")
+    click_link("an organizer, and I want to add my organization.")
+    
     expect(page).to have_content("Add Your Organization")
-    # expect(page).to have_content("Create Organization")
+  end
+
+  scenario 'logged in user can create a new org and become its organizer' do
+    user = FactoryGirl.create(:user)
+
+    sign_in_as(user)
+    visit new_organization_path
+
+    fill_in('Name', with: "Rosie\'s Place")
+    fill_in('Address', with: "889 Harrison Ave")
+    fill_in('City', with: "Boston")
+    select("Massachusetts", from: "State")
+    fill_in('Zip Code', with: "02118")
+    fill_in('Description', with: "Rosie's Place was founded in 1974 as the first
+                           women’s shelter in the United States. Our mission is
+                           to provide a safe and nurturing environment that
+                           helps poor and homeless women maintain their dignity,
+                          seek opportunity and find security in their lives. ")
+    fill_in('Phone', with: "617-442-9322")
+    fill_in('Email', with: "rosie@mailinator.com")
+    fill_in('Url', with: "http://www.rosiesplace.org/")
+    attach_file("Display photo",
+                "#{Rails.root}/spec/support/images/logo_rosiesplace.png")
+    fill_in('Delivery instructions', with: "You can drop off donations to
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.